tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -1,3 +1,73 @@
+- name: ensure network is configured
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc apply -f - <<EOF
+    apiVersion: network.openstack.org/v1beta1
+    kind: NetConfig
+    metadata:
+      name: netconfig
+    spec:
+      networks:
+      - name: CtlPlane
+        dnsDomain: ctlplane.example.com
+        subnets:
+        - name: subnet1
+          allocationRanges:
+          - end: 192.168.122.120
+            start: 192.168.122.100
+          - end: 192.168.122.200
+            start: 192.168.122.150
+          cidr: 192.168.122.0/24
+          gateway: 192.168.122.1
```

Comment:
```
{"level":"info","ts":"2023-07-28T15:32:59Z","msg":"Reconciling IPSet","controller":"openstackdataplanerole","controllerGroup":"dataplane.openstack.org","controllerKind":"OpenStackDataPlaneRole","OpenStackDataPlaneRole":{"name":"edpmadoption","namespace":"openstack"},"namespace":"openstack","name":"edpmadoption","reconcileID":"edfe14bb-90ca-46eb-ad13-e895d1d4178f","ObjectType":"*v1beta1.OpenStackDataPlaneRole","ObjectNamespace":"openstack","ObjectName":"edpmadoption"}
{"level":"error","ts":"2023-07-28T15:32:59Z","msg":"Reconciler error","controller":"openstackdataplanerole","controllerGroup":"dataplane.openstack.org","controllerKind":"OpenStackDataPlaneRole","OpenStackDataPlaneRole":{"name":"edpmadoption","namespace":"openstack"},"namespace":"openstack","name":"edpmadoption","reconcileID":"edfe14bb-90ca-46eb-ad13-e895d1d4178f","error":"admission webhook \"vipset.kb.io\" denied the request: IPSet.network.openstack.org \"standalone\" is invalid: spec.networks[0].fixedIP: Invalid value: \"38.102.83.129\": address not in IP address prefix (CIDR) 192.168.122.0/24","stacktrace":"sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.6/pkg/internal/controller/controller.go:329\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.6/pkg/internal/controller/controller.go:274\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.6/pkg/internal/controller/controller.go:235"}
```
that's why data-plane-adoption-github-rdo-centos-9-crc-single-node  works and centos-9-standalone-to-crc-data-plane-adoption breaks